docs/modules/ROOT/pages/how-to-guides/proc_managing-compliance-with-the-enterprise-contract.adoc

```diff
@@ -3,9 +3,11 @@
 [id="managing-compliance-with-the-enterprise-contract_{context}"]
 = Managing compliance with the Enterprise Contract
 
-The Enterprise Contract (EC) is an artifact verifier and customizable policy checker. You can use Enterprise Contract to keep your software supply chain secure and ensure container images comply with your organization's policies. It does this by verifying the security and provenance of builds created through {ProductName}.
+The Enterprise Contract (EC) is an artifact verifier and customizable policy checker. By default, {ProductName} adds the Enterprise Contract as an integration test to each new application. The Enterprise Contract then keeps your software supply chain secure and ensures container images comply with your organization's policies. It does this by verifying the security and provenance of builds created through {ProductName}.
```

Comment:
Perhaps we need to explore this further and it is outside the scope of this PR, but since we have stated that EC stands for Enterprise Contract, can we use the shorter form in the doc?